dp

```diff
@@ -80,16 +87,24 @@ case "$command" in
 
 # Directory to install your app to (Default: \$HOME)
 #BASE_DIR=...
+#APP_NAME=...
+
+# Log files (can't change after init-server)
+LOG_SIZE=10485760
+LOG_FILES=10
 EOF
   cp .dp/production/config .dp/staging/config
   cat << EOF > .dp/compile
 # Place compilation commands here
 EOF
+    touch Procfile
```

Comment:
`touch -a` will do a better job of leaving things unchanged if `Procfile` already exists